wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp

```diff
@@ -243,14 +222,14 @@ void CommandScheduler::Run() {
         m_impl->requirements.erase(requirement);
       }
 
-      m_impl->scheduledCommands.erase(iterator);
+      m_impl->scheduledCommands.erase(command);
```

Comment:
This is a bit suspicious to me. The old impl deleted through the iterator (I assume it's to avoid concurrent modification issues, similar to Java), but I couldn't find a way to do the same with the data structure change.